scripts/rebuild-testbench.sh

```diff
@@ -34,22 +39,93 @@ export_CC_with_afl()
     export CC=${SOF_AFL}
 }
 
+setup_xtensa_tools_build()
+{
+    BUILD_TYPE=xt
+    BUILD_TARGET=
+    BUILD_DIR_NAME=build_xt_testbench
+
+    # check needed environment variables
+    if [[ -z "${XTENSA_TOOLS_ROOT}" ]]; then
+	echo Error: XTENSA_TOOLS_ROOT need to be set.
+	exit 1
+    fi
+
+    case $BUILD_PLATFORM in
+	tgl)
+	    compiler=xt-xcc
+	    XTENSA_TOOLS_VERSION="RG-2017.8-linux"
+	    export XTENSA_CORE="cavs2x_LX6HiFi3_2017_8"
+	    ;;
+	*)
+	    echo "Error: Illegal platform $BUILD_PLATFORM"
+	    exit 1
+	    ;;
+    esac
+
+    install_bin=install/tools/$XTENSA_TOOLS_VERSION/XtensaTools/bin
+    tools_bin=$XTENSA_TOOLS_ROOT/$install_bin
+    testbench_sections="-Wl,--sections-placement $BUILD_TESTBENCH_DIR/testbench_xcc_sections.txt"
+    export CC=$tools_bin/$compiler
+    export LD=$tools_bin/xt-ld
+    export OBJDUMP=$tools_bin/xt-objdump
+    export LDFLAGS="-mlsp=sim -Wl,-LE $testbench_sections"
+}
+
+
+export_xtensa_setup()
+{
+    export_dir=$BUILD_TESTBENCH_DIR/$BUILD_DIR_NAME
+    export_script=$export_dir/xtrun_env.sh
+    xtbench=$export_dir/testbench
+    xtbench_run="XTENSA_CORE=$XTENSA_CORE \$XTENSA_TOOLS_ROOT/$install_bin/xt-run $xtbench"
+    cat <<EOFSETUP > $export_script
+export XTENSA_TOOLS_ROOT=$XTENSA_TOOLS_ROOT
+export XTENSA_CORE=$XTENSA_CORE
+XTENSA_PATH=$tools_bin
+EOFSETUP
+}
+
+show_usage()
```

Comment:
There's already a print_usage() function above, it's confusing. Maybe `testbench_usage`?